hazelcast/src/main/resources/hazelcast-config-5.4.xsd

```diff
@@ -4327,7 +4327,7 @@
                 </xs:simpleType>
             </xs:element>
         </xs:all>
-        <xs:attribute name="name" type="xs:string" default="default-tiered-store-device">
+        <xs:attribute name="name" type="xs:string" use="required">
```

Comment:
@taburet  what do we think, is this potentially a breaking change? 

cc @sumnerib